py3status/modules/sysdata.py

```diff
@@ -124,6 +127,7 @@ class Py3status:
     temp_unit = "°C"
     thresholds = [(0, "good"), (40, "degraded"), (75, "bad")]
     zone = None
+    zone_sensor = r"(?:Core 0|CPU Temp).+\+(.+).+\(.+"
```

Comment:
could we switch to a compiled regex instead?

Reply:
Hi, thanks for the review.
Changed code to use compiled regex and used self.zone_sensor instead of passing it everywhere.
One thing: i change zone_sensor type from string to compiled regex in init to keep doc tests happy.
That is not very clean... Maybe i should ignore this parameter in tests instead?